docs/getting-started/sso/keycloak.adoc

```diff
@@ -0,0 +1,14 @@
+. In order to perform SSO, you need to make sure there is at least one client is configured in Keycloak. Ensure the Keycloak client has the following settings:
```

Comment:
Just a nit pick, might be nice to have a heading here? Something like Keycloak Client Setup

Reply:
"you need to make sure there is at least one client is configured in Keycloak".  Can we remove the second "is" -> "you need to make sure there is at least one client configured in Keycloak"

Reply:
@rachael-oregan I added a heading here before, but if I do that, I will have to remove the heading from the other doc. So I leave the heading over there and removed it from this one. I think it is ok as this doc is only intended to be used internally and will be embedded in other docs.